### PR TITLE
add cd to dockerhub

### PR DESCRIPTION
**Pull request by Khorm247**

### CODE REVIEW

File: .github/workflows/deploy-to-render.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+          tags: ${{ secrets.DOCKERHUB_TAG }} # Example: username/project:latest
+          context: .
+
+  deploy:
+    name: deploy-to-render
+    runs-on: ubuntu-latest
+    needs: push-to-docker-hub
+    environment:
+      name: Capstone Project # Capstone Project name
+      url: https://neuefische.de/ # Link to deployment
```
Comment (MarcellNF) [inline .github/workflows/deploy-to-render.yml]: I don´t think that this your capstone url ;-)

**Review by MarcellNF — CHANGES_REQUESTED** [review]
LGTM so gar, but you should change the url :-)

File: .github/workflows/deploy-to-render.yml (hunk 1 above)
Comment (Khorm247) [reply .github/workflows/deploy-to-render.yml]: ty!

**Review by MaHambach — APPROVED** [review]
LGTM

**Review by MarcellNF — APPROVED** [review]
LGTM!:)

**Pull request merged**